x-pack/plugins/session_view/public/components/process_tree_alert/index.tsx

```diff
@@ -55,8 +73,9 @@ export const ProcessTreeAlert = ({
   if (!(alert.kibana && rule)) {
     return null;
   }
-
   const { name } = rule;
+  const processEventAlertCategory = category ?? 'alert';
```

Comment:
Not sure I follow whats is happening here? Is 'alert' a valid category type? Aren't the values process|file|network ?

Should it maybe default to ProcessEventAlertCategory.process ?

Reply:
The case here is alert category could be undefined or if another alert category is introduced but we don't have any alert category for that case, we would have a generic name fallback value for that case. Also, this `processEventAlertCategory` is used to get the icon associated with the alert category.